### ISSUE TITLE
Luca QR:  Replace undefined parameters from Check In  with useful Defaults 

### ISSUE THREAD

**Issue by DanAmt** [corona-warn-app/cwa-wishlist#679]
In LucaApp location QR codes fields like Location are not filled in and replaced with "<unknown>" . 
See: https://github.com/corona-warn-app/cwa-documentation/issues/741.
While this does not make a difference in warning functionality it looks ugly and has high public visibility. 

The idea  is to fill these fields with sensible defaults , like: 

_@LucaCode@
@The QR code does not provide  text  information , but warnings will work@_

or similar. This would stop people from thinking it is an error. 

Problem image see below. 
![undefined](https://user-images.githubusercontent.com/31897551/141282928-eec51e55-30b3-47b0-99d0-8de30deff1b9.jpeg)

---
Internal Tracking ID: EXPOSUREAPP-10564 (iOS)
Internal Tracking ID: EXPOSUREAPP-10625 (Android)

**Comment by viisauksena** [comment]
the luca returned string is undefined : see https://pbs.twimg.com/media/FD5XsWrXMAEYtiV?format=png

**Comment by DanAmt** [comment]
Correct, but it will be blamed on CWA, so the idea is to somehow replace the undefined with placeholders, that say "Sry, Luca has not provided"

**Comment by vaubaehn** [comment]
Would be nice if implemented quickly!

**Comment by vaubaehn** [comment]
One could think if the GUID could be displayed instead some of the `undefined`s, like

`Location ID: S897-SHJKD-9879-DAKJ`

That would people show that CWA received something that can be worked with...

**Comment by DanAmt** [comment]
I think a text would be  more user friendly. Indeed though, the users need to know their scan was successful and CWA will warn if needed. 
Example "**Luca App QR Code wurde erfolgreich verarbeitet**"  or neutral "**Fremd App QR Code wurde erfolgreich verarbeitet**"
Currently it looks like an error, which will lower trust and acceptance.

**Comment by svengabr** [comment]
Internal Tracking ID: EXPOSUREAPP-10669

@DanAmt thanks for your input! I would appreciate a text change as well.
